test/channel_permissions.js

```diff
@@ -1104,6 +1298,42 @@ describe('Commerce permissions', function() {
 		);
 	});
 
+	describe('Create message on frozen channel', function() {
```

Comment:
What is ongoing here ? duplicated ?

Reply:
It is repeated for all default channel types, hidden due to diff. 

Would be nice to dry it but not the target of this PR, of course.